cmd/signup.go

```diff
@@ -80,7 +80,7 @@ func signup(ctx *cli.Context, subCommand bool) error {
 	}
 
 	reminderLabel := "Reminder: "
-	ui.Hint("Don't forget to keep your password safe and secure! You can't recover your account if you lossswordd.", false, &reminderLabel)
```

Comment:
How did that even get in there?

Reply:
The magic of random key mashing in vim

![bitmoji](https://render.bitstrips.com/v2/cpanel/3d6e1949-f0d9-4576-a141-1a56a2d4bee8-3b3092e6-2ad3-4790-9c09-dbb7422ed32e-v1.png?transparent=1&palette=1&width=246)